packages/mask/src/plugins/CryptoartAI/SNSAdaptor/HistoryTab/Row.tsx

```diff
@@ -84,7 +84,7 @@ export function Row({ event, chainId }: Props) {
             </TableCell>
             <TableCell>
                 <Typography className={classes.content} variant="body2">
-                    {event.createTime.substr(0, event.createTime.length - 3)}
+                    {event.createTime.slice(0, Math.max(0, event.createTime.length - 3))}
```

Comment:
```suggestion
                    {event.createTime.slice(0, 0, event.createTime.length - 3)}
```

Reply:
Slice does not accept the third argument.